mapmssql2008.c

```diff
@@ -2389,7 +2389,10 @@ int process_node(layerObj* layer, expressionObj *filter)
     case MS_TOKEN_LITERAL_BOOLEAN:
     case MS_TOKEN_LITERAL_NUMBER:
       strtmpl = "%lf";
-      snippet = (char *) msSmallMalloc(strlen(strtmpl) + 16);
+      if (layerinfo->current_node->tokenval.dblval>MS_MAX_DOUBLE || layerinfo->current_node->tokenval.dblval<MS_MIN_DOUBLE)
```

Comment:
Couldn't we have a char* msPrintLf(double) function that could be reused in all those multiple locations ?